Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -22,5 +52,150 @@ namespace Baikal
                                                    height,
                                                    input_channels);
         }
+
+        MLDenoiser::MLDenoiser(CLWContext context, Inference::Ptr inference, MLDenoiserInputs inputs)
+                   : m_inference(std::move(inference))
+        {
+            m_context = std::move(std::make_unique<CLWContext>(context));
+            m_primitives = std::move(std::make_unique<CLWParallelPrimitives>(context));
+
+            auto shape = m_inference->GetInputShape();
+            auto width = std::get<0>(shape);
+            auto height = std::get<1>(shape);
+            auto chanels_num = std::get<2>(shape);
+
+            size_t elems_count = sizeof(Tensor::ValueType) * width * height * chanels_num;
```

Comment:
I think it's not _elem_count_, but may be _bytes_...

Reply:
elem_count is CLW terminology for this type of buffers